wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+              sortWords(words!);
+            ">
+            <strong>Common Word</strong
+            ><v-icon :icon="getCommonWordIcon()"></v-icon>
+          </th>
+          <th class="text-center text-h8"><strong>Edit</strong></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(word, index) in words">
+          <td class="text-center text-h8">
+            {{ word.word }}
+          </td>
+          <td class="text-center text-h8">
+            {{ word.isCommonWord ? 'Yes' : 'No' }}
+          </td>
+          <td class="text-center text-h8">
+            <v-row class="mt-0">
+              <v-col cols="3">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="tokenService?.isLoggedIn()">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-switch
+                        :color="tokenService?.isLoggedIn() ? 'primary' : ''"
+                        :disabled="!tokenService?.isLoggedIn()"
+                        v-model="word.isCommonWord"
+                        @update:modelValue="
+                          isCommonWordChanged[index] =
+                            !isCommonWordChanged[index]
+                        "></v-switch>
+                    </div>
+                  </template>
+                  <span>You must be logged in to change common word flag</span>
+                </v-tooltip>
+              </v-col>
+              <v-col class="mt-2" cols="4">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="isOlderThanTwentyOne && isMotU">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-btn
+                        :color="isOlderThanTwentyOne && isMotU ? 'error' : ''"
+                        :disabled="!(isOlderThanTwentyOne && isMotU)"
+                        :variant="
+                          isOlderThanTwentyOne && isMotU ? 'outlined' : 'plain'
+                        "
+                        icon="mdi-trash-can-outline"
+                        size="35"
+                        @click="deleteWord(word)"></v-btn>
+                    </div>
+                  </template>
+                  <span
+                    >You must be Master of the Universe and 21 or older to add
+                    words</span
+                  >
+                </v-tooltip>
+              </v-col>
+              <v-col class="mt-2" cols="4">
+                <v-btn
+                  :variant="isCommonWordChanged[index] ? 'elevated' : 'plain'"
+                  @click="
+                    isCommonWordChanged[index] = false;
+                    saveCommonWordFlag(word.word, word.isCommonWord);
+                  "
+                  :color="isCommonWordChanged[index] ? 'primary' : ''">

```

Comment:
I strongly suggest changing the way you are formatting the style of button. The plain variant of the button makes it hard to tell if the button is disabled or enabled
![botton save](https://github.com/IntelliTect-Samples/EWU-CSCD379-2024-Spring/assets/114778753/4286d4b7-32de-40e0-af03-49b256b417a6)

Image for reference from light mode.

Reply:
will work on that